Client/Frontend/Browser/Toolbars/UrlBar/TabLocationView.swift

```diff
@@ -176,17 +185,36 @@ class TabLocationView: UIView {
         addGestureRecognizer(longPressRecognizer)
         addGestureRecognizer(tapRecognizer)
         
+        #if NO_REWARDS
         [readerModeButton, separatorLine, shieldsButton].forEach {
             $0.setContentCompressionResistancePriority(.defaultHigh, for: .horizontal)
             $0.setContentHuggingPriority(.defaultHigh, for: .horizontal)
         }
         
-        // Make tappable area bigger than the icon.
+        // Increasing tappable area of buttons, so user won't by mistake tap on address bar.
+        // Creating a stack view for buttons with 0 spacing.
+        // Extra space is added by `contentEdgeInsets`
+        let buttonsStackview = UIStackView(arrangedSubviews: [shieldsButton])
+        
         shieldsButton.contentEdgeInsets = UIEdgeInsets(top: 6, left: 6, bottom: 6, right: 6)
+        #else
+        [readerModeButton, separatorLine, shieldsButton, rewardsButton].forEach {
+            $0.setContentCompressionResistancePriority(.defaultHigh, for: .horizontal)
+            $0.setContentHuggingPriority(.defaultHigh, for: .horizontal)
+        }
+        
+        // Increasing tappable area of buttons, so user won't by mistake tap on address bar.
+        // Creating a stack view for buttons with 0 spacing.
+        // Extra space is added by `contentEdgeInsets`
+        let buttonsStackview = UIStackView(arrangedSubviews: [shieldsButton, rewardsButton])
+        
+        shieldsButton.contentEdgeInsets = UIEdgeInsets(top: 4, left: 8, bottom: 4, right: 7.5)
+        rewardsButton.contentEdgeInsets = UIEdgeInsets(top: 4, left: 7.5, bottom: 4, right: 13)
```

Comment:
`contentEdgeInsets` should be uniform otherwise we get a non-centred popover arrow.  Consider the same solution as https://github.com/brave/brave-ios/pull/1197